src/main.ts

```diff
@@ -8,4 +8,17 @@ if (environment.production) {
   enableProdMode();
 }
 
-platformBrowserDynamic().bootstrapModule(AppModule);
+// use the require method provided by webpack
+declare const require;
+// we use the webpack raw-loader to return the content as a string
+
+const translations = require(`raw-loader!./i18n/messages.ne.xlf`);
```

Comment:
why are these hardcoded

Reply:
just to check it works or not as we can make dynamic  later on as per required language translation.

Reply:
all these are served when you load ng serve with locale mentioned as AOT has defined so why do we need to mention all files and locale here